### ISSUE TITLE
No Icons in VSCode Insiders - 2016-08-22T08:10:56.602Z

### ISSUE THREAD

**Issue by brakmic** [robertohuertasm/vscode-icons#176]
It seems the previously removed bug is back again: https://github.com/robertohuertasm/vscode-icons/issues/173

My current Insiders version of VSCode:
```
Version 1.5.0-insider
Commit 435df14a684df718c5ec523f9d674e63d1662c37
Date 2016-08-22T08:10:56.602Z
Shell 0.37.6
Renderer 49.0.2623.75
Node 5.10.0
```

**Comment by Tekbr** [comment]
Same.

**Comment by thiagoaos** [comment]
Same here. Only folder icon appear.

**Comment by kumarharsh** [comment]
Seems like the latest update to this plugin broke everything. No icons for me too on Windows 10

**Comment by robertohuertasm** [comment]
I guess you all have no errors in your dev consoles, haven't you?

**Comment by brakmic** [comment]
Sadly, I couldn't find any errors in debug console.

**Comment by robertohuertasm** [comment]
Ok, thanks @brakmic. For sure, something has changed again. I'll take a look asap and let you know.

**Comment by robertohuertasm** [comment]
I think they're starting to deploy new features to support icons in an official way according to [6fe06f0](https://github.com/Microsoft/vscode/blob/34377a9e99cf8f4ddca6be4e70c7c7333b2cd0ca/src/vs/workbench/parts/files/browser/views/explorerViewer.ts#L320) and [#10600](https://github.com/Microsoft/vscode/issues/10600)

I'll have to take a deep look at the new code and see what can be done. Sorry for the inconveniences :disappointed:

**Comment by brakmic** [comment]
No problem. 

It'll be a hard-knock life without your cool icons but I think most of us will survive :smiley: 

Thanks :+1:

**Comment by robertohuertasm** [comment]
:grinning: You can also use the stable version... :grin: 
Thank you for your kind support! :wink:

**Comment by egamma** [comment]
@robertohuertasm you are correct given your great work we started to work on official support for icon extensions. Official support will avoid issues like this one. 

I hope this doesn't come as a surprise since we had this on our August plan (https://github.com/Microsoft/vscode/issues/10145). 

I've created issue #177 to start the discussion and I've CCed @aeschli the developer of this support.

**Comment by robertohuertasm** [comment]
Thanks @egamma for you answer! :+1: 

I knew that you were working on this so it's not really a surprise but I didn't know anything about the specifics of the implementation. 

Of course, I'll try to integrate the extension with the new API in order to get the same functionalities that we currently have in an official way! :wink:

**Comment by robertohuertasm** [comment]
@brakmic I've just published 1.7.2 with support for new api changes so insiders build should be working again with all the features. Still doing it the hacky way though. I'll have to think about the integration process and see how we manage it.
